app/css/question.css

```diff
@@ -1,12 +1,35 @@
-#question_field {
+.QuestionForm {
+    border-top: 1px solid #e7e7e7;
+}
+.QuestionField {
     border: 0;
-    max-height: 100px;
+    color: #000;
+    font-size: 17px;
+    max-height: 142px;
```

Comment:
I'll add a comment. But that's the 5 line max size that Helen specified for when the field expands.